tools/cell_census_builder/tests/test_manifest.py

```diff
@@ -0,0 +1,113 @@
+import io
```

Comment:
you might consider a test that loads from manifest, where the manifest has a duplicate, thereby testing the `dedup_datasets` code.

This is not critical, but I added the dedup code when I accidentally created a file-based manifest with the same dataset ID!

Reply:
👍 will add.

Reply:
Added.